sources/lib/api/gcp/bigquery/_udf.py

```diff
@@ -13,34 +13,27 @@
 """Google Cloud Platform library - BigQuery UDF Functionality."""
 
 import json
+import gcp.storage
 
 
 class FunctionCall(object):
```

Comment:
Actually, if we're supporting calling UDFs in only SQL then we don't need UDF.\_\_call\_\_, nor do we need FunctionEvaluation or FunctionCall; they can all go.